doc/manual/parameters.tex

```diff
@@ -14974,7 +14974,7 @@ \subsection{Parameters in section \tt Material model/Visco Plastic}
 {\it Default:} 293.
 
 
-{\it Description:} For calculating density by thermal expansivity. Units: $\si{K}$
+{\it Description:} The reference temperature $T_0$. Units: $\si{K}$.
```

Comment:
Above you remove the dollar signs -- here you don't. Intentional?

Reply:
This one wasn't my change - parameters.tex wasn't fully up-to-date
I'll modify Material model/Visco Plastic (along with other files) in the next PR.

Reply:
Ok, all updated and ready to merge. I'll get to work on the next PR :)